markup5ever/build.rs

```diff
@@ -89,27 +83,16 @@ fn main() {
     writeln!(generated, "}}").unwrap();
 }
 
-fn named_entities_to_phf(from: &Path, to: &Path) {
-    // A struct matching the entries in entities.json.
-    #[derive(Deserialize, Debug)]
-    struct CharRef {
-        codepoints: Vec<u32>,
-        //characters: String,  // Present in the file but we don't need it
-    }
+include!("entities.rs");
```

Comment:
No need to use include! here; we can use `mod entities;` and refer to `entities::NAMED_ENTITIES`.

Reply:
TIL that one can use mod in build.rs :smiley:  Updated.